di/di.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2020 PingCAP-QE libs Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package di
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"log"
+	"strings"
+	"time"
+)
+
+// DI constants
+const (
+	CRITICAL_DI = 10.0
+	MAJOR_DI    = 3.0
+	MODERATE_DI = 1.0
+	MINOR_DI    = 0.1
+)
+
+// MySQL config
+const MYSQL_QUERY_TIMEOUT = 5 * time.Second
+
+// Issue struct
+type Issue struct {
+	ID           uint
+	Number       int
+	RepositoryID int
+	Closed       bool
+	ClosedAt     time.Time
+	CreatedAt    time.Time
+	Title        string
+	Label        map[string][]string
+}
+
+// calculateDI returns total DI of specified issues
+func calculateDI(issues []Issue) float64 {
```

Comment:
How about extract a function such as getDIofIssue()?  We may calculate much kind of DI, such as a version's DI, and Total DI